ale_linters/llvm/llc.vim

```diff
@@ -0,0 +1,31 @@
+" Author: rhysd <https://rhysd.github.io>
+" Description: Support for checking LLVM IR with llc
+
+call ale#Set('llvm_llc_command', 'llc')
+
+function! ale_linters#llvm#llc#GetCommand(buffer) abort
+    return ale#Var(a:buffer, 'llvm_llc_command')
+    \   . ' -filetype=null -o='
+    \   . ale#Escape(g:ale#util#nul_file)
```

Comment:
I just merged a pull request which used `/dev/null` on Windows for elm. Do you know of `nul` will actually work for `-o` on Windows? I wouldn't be surprised if yet another tool has special code for exactly the string `/dev/null`, but doesn't know what to do with `nul`.

Reply:
I'm usually not using Windows. But I dug LLVM source code a bit. Filesystem support is implemented here and llc seems to use this.

https://github.com/llvm-mirror/llvm/blob/master/include/llvm/Support/FileSystem.h

As long as I read the code, LLVM does not handle `/dev/null` specially. So I think `nul` is available but `/dev/null` is unavailable on Windows.